data/governments.txt

```diff
@@ -546,6 +546,20 @@ government "Hai (Unfettered)"
 	"hostile hail" "hostile unfettered"
 	"hostile disabled hail" "hostile disabled unfettered"
 
+government "Hai (Unfettered Challenger)"
+	"display name" "Unfettered Heshi"
```

Comment:
Any lore on that name? What does it mean? (And where did it come from, your head or something else!)

Reply:
The lore for it is in the Unfettered campaign PR:
`	"The Heshi is a great predator which rules the seas on our homeworld. It is swift and nimble, so you do not see it coming, and when you do, it is already too late. They are not very big, however, so they often hunt in groups. Your people have described it as looking like a 'sea scorpion', I believe, and rumour has it you acquired one from Si-Ra."`

Reply:
Interesting, thanks!